### PR TITLE
Updated Readme and Pyproject with links to github pages documentation

### PR DESCRIPTION
**Pull request by SWARUP-Selvaraj**
- Readme updated with badges and links to documentation
- pyproject updated with documentation link and version information

### CODE REVIEW

**Comment by connorferster** [comment]
Many thanks!

File: pyproject.toml
Comment (connorferster) [inline pyproject.toml:1]: @SWARUP-Selvaraj Thank you for this pull request!

One small change needed: please remove the version specifier from pyproject.toml (it is dynamically generated by flit).

Comment (SWARUP-Selvaraj) [reply pyproject.toml:1]: Hope [this Commit](https://github.com/connorferster/forallpeople/commit/c2826e433ceb8def1f06a6bbe9b7205d00693d42) incorporates your comment😊

I never used flit. Will give it a try and let you know👍

**Comment by connorferster** [comment]
Hi @SWARUP-Selvaraj,

Thank you for the PR and the updates. Much appreciated. While I was reviewing I fixed a small syntax error but then realized that my pyproject.toml file is _horribly_ out of date. So, I copied your changes over and put them into a new pyproject.toml fie.

**Comment by connorferster** [comment]
Let's merge this!

**Pull request merged**